Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -72,6 +75,7 @@ class DataDecoderTask
     mDebug = ic.options().get<bool>("debug");
     mCheckROFs = ic.options().get<bool>("check-rofs");
     mDummyROFs = ic.options().get<bool>("dummy-rofs");
+    mNHBPerTF = ic.options().get<int>("nhb-per-tf");
```

Comment:
Can't you simply get this information from the HBFUtils ConfigurableParam (https://github.com/AliceO2Group/AliceO2/blob/d26801ce514a23b839e30e50f1a67aef6b46a38c/Detectors/Raw/include/DetectorsRaw/HBFUtils.h#L128) and then use the command line syntax `--configKeyValues "HBFUtils.nHBFPerTF=128` to set it ? (like what's done in the [digitization](https://github.com/AliceO2Group/AliceO2/blob/dev/doc/DetectorSimulation.md) for instance)  

Actually same remark/question for the first orbit ?

Reply:
Yes, good point, I was actually unsure of how to set this from the command line...

However, for the first orbit in the run we cannot rely on the command line, since it can change from one run to another in a random way.